spec/bing_translator_spec.rb

```diff
@@ -4,33 +4,18 @@
 
 require File.join(File.dirname(__FILE__), '..', 'lib', 'bing_translator')
 
-# Log all the tasks output, keeping RSpec interface clean while running
```

Comment:
Alright. I'm not sure why it was there in the first place, so I'm happy to delete unnecessary code unless @relrod speaks up.